src/test/java/org/elasticsearch/index/query/BaseQueryTestCase.java

```diff
@@ -64,8 +64,8 @@
 public abstract class BaseQueryTestCase<QB extends BaseQueryBuilder & Streamable> extends ElasticsearchTestCase {
 
     private static Injector injector;
-    private static IndexQueryParserService queryParserService;
-    private static Index index;
+    protected static IndexQueryParserService queryParserService;
+    protected static Index index;
```

Comment:
I think those two can stay private, or do you need them in the concrete test classes? They should also be accesible via the context you get from createContext() if you need them.